specs/global/interfaces.json

```diff
@@ -50,14 +50,11 @@
             { "$ref": "#/HashBearing" }
         ]
     },
-    "HashedSnakList": {
-        "type": "array",
-        "items": { "$ref": "#/HashedSnak" }
-    },
-    "HashedSnakMap": {
+    "HashedSnaks": {
```

Comment:
I would even go further here and say that a `HashedSnak` = `Qualifier` (As we have not other occurences of hashed snaks in this API, as opposed to the action api, we go with the Special:EntityData JSON response format here). But yeah, this definitely needs cleaning up, as there is also this "HashBearing" above.